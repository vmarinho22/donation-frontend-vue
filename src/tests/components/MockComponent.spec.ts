import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import MockComponent from './MockComponent.vue'

describe('ToggleTheme', () => {
  it('renders the component', () => {
    const wrapper = mount(MockComponent)
    expect(wrapper.html()).toMatchSnapshot();
  })
})