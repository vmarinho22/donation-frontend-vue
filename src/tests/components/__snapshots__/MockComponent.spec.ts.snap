// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ToggleTheme > renders the component 1`] = `"<p>Mock component</p>"`;
